src/audio/src/src_common.c

```diff
@@ -593,6 +594,60 @@ int src_param_set(struct comp_dev *dev, struct comp_data *cd)
 	return 0;
 }
 
+static void debug_print(struct comp_dev *dev, const char *name, const struct src_stage *s)
+{
+	comp_info(dev, "%s: %d, %d, %d, %d, %d, %d, %d, %d, %d, %p", name,
+		  s->idm, s->odm, s->num_of_subfilters, s->subfilter_length,
+		  s->filter_length, s->blk_in, s->blk_out, s->halfband, s->shift,
+		  s->coefs);
+}
+
+int src_allocate_copy_stages(struct comp_dev *dev, struct src_param *prm,
+			     const struct src_stage *stage_src1,
+			     const struct src_stage *stage_src2)
+{
+	struct src_stage *stage_dst1;
```

Comment:
maybe just use an array as `struct src_stage *stage_dst` and then `stage_dst[0].coefs` etc. but up to you too, just an idea

Reply:
I don't bother with the change, not yet anyway. I have a hunch that there will be more rounds coming.

Reply:
Can we put the ifdef FAST_GET at the function level i.e. if FAST_GET=n then ```src_allocate_copy_stages()``` does what it does today, if =y then it does the fast get logic